src/App.tsx

```diff
@@ -1,7 +1,19 @@
-import { useState } from "react";
+import React, { useEffect, useState } from "react";
+import { UserContext } from "./contexts/UserContext";
+import { useLocalStorage } from "./hooks/useLocalStorage";
+import { User } from "./interfaces/User";
+import { AppRouter } from "./routes/AppRouter";
+import { STORAGE_KEY } from "./utils/constants";
 
-function App() {
-  return <div></div>;
-}
+const App = () => {
+  const [storedUser, setStoredUser] = useLocalStorage(STORAGE_KEY.USER, "");
+  const [loggedUser, setLoggedUser] = useState<User | false>(storedUser);
```

Comment:
I think null would be better than using false, I recommend using false/true only for boolean values, but here user is an object

Reply:
Good point Gerardo, thanks for the feedback! This was just fixed